juju/juju
# Move all EnableHA functions out of addmachine.go

## Description of change

Instead of clumping them all together in state_test.go and
addmachine.go, pull them out into their own file enableha.go and
enableha_test.go. Should be more understandable when they are grouped
together like that.

This simply moves the code, it makes no changes to the individual lines.

## QA steps

Nothing externally visible should change. You can run just these tests with:
```
$ cd state
$ go test -check.v -check.f EnableHA
```

## Documentation changes

None.

## Bug reference

None.
